Solutions/Votiro/Analytic Rules/VotiroFileBlockedFromConnector.yaml

```diff
@@ -0,0 +1,28 @@
+id: 17bf3780-ae0d-4cd9-a884-5df8b687f3f5
+name: Votiro - File Blocked from Connector
+description: |
+ 'If the syslog message includes-Blocked under  -Sanitization result-field  and false under -password protected- field and null under -from- field  create an alert with the following message:[file name] with hash [file hash] that was sent from connector [connector name] was blocked by Votiro due to Policy [policy name], see more detail in the following link [incident url]'
+severity: Low
+requiredDataConnectors:
+  - connectorId: Votiro
+    dataTypes:
+     - CommonSecurityLog 
+queryFrequency: 10m
+queryPeriod: 10m
+triggerOperator: gt
+triggerThreshold: 0
+query: let Votiro_view  = view () {
+  VotiroEvents
+   | where SanitizationResult == "Blocked" and PasswordProtected == "false" and From == "null"
+   | extend FileWithConnectorDetails = strcat_delim(' ', SrcFileName, 'with the hash', SrcFileSHA256, 'that was sent from connector', ConnectorName)};Votiro_view
```

Comment:
Use parse-where function to ensure only required records are captured here.

Reply:
hi @devikamehra,
This is a business rule, we need to get all the coloumns we used in "where" condtion along with "parse-where". We can't use more than one coloumn in parse-where operator. So we have made changes accordingly.
If you know anything else please suggest.
Thanks,
Sambhrant.